src/lua/bindings/network.cpp

```diff
@@ -227,6 +259,8 @@ namespace lua::network
 
 		auto ns = state["network"].get_or_create<sol::table>();
 
+		ns["get_player_rank"]                          = get_player_rank;
+		ns["get_player_rp"]                            = get_player_rp;
```

Comment:
Can you put this in chronological order that they appear in the class?

Reply:
aren't they already in chronological order?

Reply:
done